screens/LoadingScreen.js

```diff
@@ -13,8 +13,27 @@ export default class LoadingScreen extends React.Component {
     this.props.navigation.navigate('ShopSearch')
     }
 
+fetchData = () => {
```

Comment:
make this into an `async` function if you want to have the ability to return `acceptedValue`